files/vacuum_full_tables.sh

```diff
@@ -1,39 +1,43 @@
 #!/bin/bash
 
-if [ "$1" = "" ]; then
-  echo "Usage: $0 <(facts, catalogs, or other) tables to VACUUM FULL> "
-  exit
-fi
+usage() {
+  cat <<EOF
+echo "usage: $0 <table>"
+where <table> is one of: facts catalogs other
```

Comment:
This change is inaccurate, as the parameter os a set of tables, rather than a single table, as per the case statement below. Also, the sleep parameter, $2, is not documented.